src/wp-includes/option.php

```diff
@@ -277,6 +277,19 @@ function form_option( $option ) {
 function wp_load_alloptions( $force_cache = false ) {
 	global $wpdb;
 
+	/**
+	 * This filter allows the wp load all options function to be shortcut.
+	 *
+	 * @since 6.1.0
+	 *
+	 * @param null $pre if an array is returned then the wp_load_alloptions will be shortcuted and the array returned.
```

Comment:
I think the param doc needs to update as it also uses `array`
```
* @param array|null $pre if an array is returned then the wp_load_alloptions will be shortcuted and the array returned. Default null.
```

Reply:
good spot fixed